icarus_backend/middleware/DisableCSRF.py

```diff
@@ -0,0 +1,6 @@
+from django.utils.deprecation import MiddlewareMixin
+
+
+class DisableCSRF(MiddlewareMixin):
```

Comment:
okay there is actually a way to disable csrf reqs per method but that should be fine as well

Reply:
yeah I was using the decorator before but I thought this would be less code.